lectures/inequality.md

```diff
@@ -1084,7 +1084,7 @@ df_income_wealth.head(n=4)
 We will focus on wealth variable `n_wealth` to compute a Gini coefficient for the year 1990.
```

Comment:
@longye-tian can we either change `1990` in the text to `2016` or the otherway around based on the correct context?